dashboard/app/controllers/certificate_images_controller.rb

```diff
@@ -27,8 +27,8 @@ def show
     # code tutorial.
     course_name = recognized_course_name?(data['course']) ? data['course'] : 'hourofcode'
 
-    course_version = CurriculumHelper.find_matching_course_version(course_name)
-    course_title = course_version&.localized_title
+    unit_or_unit_group = CurriculumHelper.find_matching_unit_or_unit_group(course_name)
+    course_title = unit_or_unit_group&.localized_title
```

Comment:
I assume localized_title still works on a unit_or_unit_group and not just course_version?

Reply:
Yes! course_version actually [delegates that function](https://github.com/code-dot-org/code-dot-org/blob/e4e4c475df0449dd304fa68d20cf87ee1289fd4e/dashboard/app/models/course_version.rb#L62) to either the unit or the unit group anyway